src/lib/import-export/import/importers/importer.ts

```diff
@@ -71,9 +71,45 @@ export abstract class BaseImporter extends EventEmitter implements Importer {
 			}
 		}
 
+		this.fixSiteUrl( siteId );
 		this.emit( ImportEvents.IMPORT_DATABASE_COMPLETE );
 	}
 
+	private async fixSiteUrl( siteId: string ) {
+		const server = SiteServer.get( siteId );
+		if ( ! server ) {
+			throw new Error( 'Site not found.' );
+		}
+
+		// Step 1: Fetch the current site URL
+		const { stdout: currentSiteUrl } = await server.executeWpCliCommand(
+			`option get siteurl --require=/tmp/sqlite-command/command.php`
```

Comment:
How a SQLite command help us here?

Reply:
Yep, I think this is not needed as the SQLite command is only needed when importing.

Reply:
Yes, sorry it was leftover. Removed!